RiskChanges/computeExposure.py

```diff
@@ -119,15 +117,13 @@ def pointExposure(ear,haz,expid,Ear_Table_PK):
 
 
 def ComputeExposure(con,earid,hazid,expid,**kwargs):
-    try:
-        is_aggregated=kwargs['is_aggregated']
-        onlyaggregated=kwargs['only_aggregated']
-        adminid=kwargs['adminunit_id']
-    except:
-        is_aggregated= False
-        onlyaggregated= False
+    is_aggregated = kwargs.get('is_aggregated', False)
+    onlyaggregated = kwargs.get('only_aggregated', False)
+    adminid = kwargs.get('adminunit_id', None)
+    haz_file = kwargs.get('haz_file', None)
+    
     ear=readear(con,earid)
-    haz=readhaz(con,hazid)
+    haz=readhaz(con,hazid, haz_file)
```

Comment:
Actually, what I faced it, if I got the file from DB, it throws me a 404 not found error. But If I manually pass the file, it worked. In Django, it stored the relative path but using the relative path it always shows a 404 error. That's why I am getting the absolute path using Django like this `haz_file = exposure.hazard_index.file.path` and passing it manually. SOrry for the redundancy but I don't know another way.

Reply:
do you know the absolute path before relative path start in django? like /home/data/ could be starting dir and then the django relative path can be django/test1/tek.tif, if you know this path, is it okay to hard code it and join path in later stage? for now this solution is okay but for future reference we could do that?

Reply:
Actually if we hardcoded on library, it will work only on  the production not in development (from windows development environment). 

We can easily get the absolute path using Django command `exposure.hazard_index.file.path`.  Which solve the dev and production environment conflict.

Reply:
okay then i think small redundancy like this will make less issues than that of hard coding the path. so you can just use that way.

Reply:
Ok sure. Thank you